components/builder-depot/src/server.rs

```diff
@@ -466,6 +466,11 @@ pub fn origin_member_delete(req: &mut Request) -> IronResult<Response> {
         None => return Ok(Response::with(status::BadRequest)),
     };
 
+    // Do not allow the owner to be removed which would orphan the origin
+    if account_name == session.get_name() {
+        return Ok(Response::with(status::BadRequest));
+    }
+
```

Comment:
This is just preventing someone from removing themselves, though right?  If you own the origin and I submit a request to remove you, it would still succeed.

Reply:
but only owners can remove anyone. Above this is a check to ensure you are an owner

Reply:
Ohhh, no my mistake.  Because this endpoint only works for the owner, right?.  I missed the code above on line 460.

Reply:
so only an owner can remove an owner. maybe I should have the comment make that more clear

Reply:
heh. yeah you would never know given the code window in this PR